tests/build.snapshot.js

```diff
@@ -7,7 +7,8 @@ const smallBuildCommons = {
 	'polyfills.js': { size: 4620 },
 	'polyfills.js.map': { size: 31760 },
 	'favicon.ico': { size: 15086 },
-	'sw.js': { size: 3330 },
+	'sw.js': { size: 1628 },
+	'workbox-sw.prod.v1.1.0.js': {size: 47026},
```

Comment:
Hey folks. Let's hold off on merging this change for a lil' bit. 47KB is way too large a bump here. We're actively working on bundle trimming to hit parity with our older builds and will reopen once we're ready.